src/containers/pages/MDAPoint/ClientListView/index.tsx

```diff
@@ -80,25 +81,31 @@ export const buildListViewData: (rowData: File[]) => ReactNode[][] | undefined =
 
 export const ClientListView = (props: ClientListViewProps & RouteComponentProps) => {
   const { location, files, clientLabel } = props;
+
+  const [loading, setLoading] = useState(false);
+
+  // update loading status
+  const isLoading = (status: boolean) => setLoading(status);
```

Comment:
Just curious here; why not pass `setLoading` itself as the `isLoading` param, without having to wrap it in another function